apps/condo/lang/ru/messages/CUSTOMER_IMPORTANT_NOTE_TYPE/default.njk

```diff
@@ -0,0 +1,2 @@
+Название: {{ message.meta.organization.name }},
```

Comment:
so we have at least two sources of truth for translations...

Reply:
I didn't understand which two sources you mean.
`njk` templates are for complex messages including formatting.
`json` files are for translating simple strings.